tests/indent/hcl_spec.lua

```diff
@@ -0,0 +1,20 @@
+local Runner = require("tests.indent.common").Runner
+--local XFAIL = require("tests.indent.common").XFAIL
+
+local run = Runner:new(it, "tests/indent/hcl", {
+  tabstop = 2,
+  shiftwidth = 2,
+  expandtab = true,
+})
+
+describe("indent HCL:", function()
+  describe("whole file:", function()
+    run:whole_file(".", {
+      expected_failures = {},
+    })
+  end)
+
+  describe("new line:", function()
+    run:new_line("no-indent-after-brace.tf", { on_line = 4, text = "# Wow, no indent here please", indent = 2 })
```

Comment:
indent should be 0 here right?

Reply:
yes, the 2 was just to try to make it failing.

Reply:
ah ok, fair enough, sorry

Reply:
No, you were right. I forgot to remove that. Thank you!